6_Xenium_sample_analysis/ pre-designed_add-on_panel/xenium_add_on_creation_of_anndata.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<p>I would add here already a QC step to see how good/bad the data looks like. See here: <a href="https://squidpy.readthedocs.io/en/stable/notebooks/tutorials/tutorial_xenium.html#calculate-quality-control-metrics" target="_blank">https://squidpy.readthedocs.io/en/stable/notebooks/tutorials/tutorial_xenium.html#calculate-quality-control-metrics</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>